Sources/NIOCore/EventLoop.swift

```diff
@@ -551,6 +551,94 @@ public struct TimeAmount: Hashable, Sendable {
     }
 }
 
+/// Contains the logic for parsing time amounts from strings,
+/// and printing pretty strings to represent time amounts.
+extension TimeAmount: CustomStringConvertible {
+
+    /// Errors thrown when parsint a TimeAmount from a string
+    public enum ValidationError: Error, Equatable {
```

Comment:
I'm not sure it's a great idea to introduce a new public error type. Can I get away with re-using something?

Reply:
I doubt that anybody would really _catch_ that error, so we could also just make this an `internal` error type. That way you can still throw & print it but NIO wouldn't be forced into the API.